buffer-tests/src/test/java/io/netty/buffer/api/tests/BufferRefTest.java

```diff
@@ -99,11 +102,13 @@ public void mustCloseOwnedBufferWhenReplacedFromSend() {
                 ref.replace(buf.send()); // Pass replacement via send().
             }
 
-            assertThrows(IllegalStateException.class, () -> orig.get().writeInt(32));
+            final RuntimeException w1e = assertThrows(RuntimeException.class, () -> orig.get().writeInt(32));
+            assertTrue(w1e instanceof IllegalStateException || w1e instanceof UnsupportedOperationException);
             ref.contents().writeInt(42);
             assertThat(ref.contents().readInt()).isEqualTo(42);
             ref.close();
-            assertThrows(IllegalStateException.class, () -> ref.contents().writeInt(32));
+            final RuntimeException w2e = assertThrows(RuntimeException.class, () -> ref.contents().writeInt(32));
+            assertTrue(w2e instanceof IllegalStateException || w2e instanceof UnsupportedOperationException);
```

Comment:
Same for these two. Looks like this test got duplicated somehow (the one above this one is the duplicate, which can be removed).